crates/bevy_app/src/app_builder.rs

```diff
@@ -290,6 +312,26 @@ impl AppBuilder {
             .add_system_set_to_stage(stage, State::<T>::get_driver())
     }
 
+    /// Adds utility stages to the [`Schedule`], giving it a standardized structure.
+    ///
+    /// Adding those stages is necessary to make some core engine features work, like
+    /// adding systems without specifying a stage, or registering events. This is however
+    /// done by default by calling `AppBuilder::default`, which is in turn called by
+    /// [`App::build`].
+    ///
+    /// # The stages
+    ///
+    /// All the added stages, with the exception of the startup stage, run every time the
+    /// schedule is invoked. The most relevant stages are the following, in order of execution:
```

Comment:
I would very much like to see a complete enumeration here: this is important information and there's only two missing stages IIRC.

Reply:
Done. I also listed the startup stages.